README.md

```diff
@@ -148,6 +149,43 @@ APP_SETTINGS_NAME=my-really-awesome-app
 
 The environment variable has the precedence over the configuration file.
 
+#### Complex values representation
+
+Since `yaml` data type support is much richer than environment variables we have to do extra care if we want to
+override `yaml` complex data types such as `list` and `dictionary`.
```

Comment:
```suggestion
Since `yaml` data type support is much richer than environment variables we have to take extra care if we want to
override `yaml` complex data types such as `list` and `dictionary`.
```